assets/javascripts/kitten/components/buttons/tag-button/index.js

```diff
@@ -1,16 +1,16 @@
-import React, { Component } from 'react'
+import React from 'react'
 import PropTypes from 'prop-types'
-import styled, { css } from 'styled-components'
+import styled from 'styled-components'
 import { pxToRem, stepToRem } from '../../../helpers/utils/typography'
 import TYPOGRAPHY from '../../../constants/typography-config'
 import COLORS from '../../../constants/colors-config'
+import classNames from 'classnames'
 
 const StyledTagButton = styled(
-  ({ selected, icon, tiny, big, huge, modifier, ...others }) => (
+  ({ selected, icon, tiny, big, modifier, ...others }) => (
```

Comment:
il me semble que tu n'as même pas besoin de supprimer les props comme ça, StyledComponent le fait déjà si tu fais un simple 

```
styled.button``
```

```suggestion
  ({ selected, icon, tiny, big, modifier, ...others }) => (
```